configs/default/JHEF-JHEF411.config

```diff
@@ -0,0 +1,235 @@
+# Betaflight / STM32F411 (S411) 4.2.0 Jun 14 2020 / 03:04:43 (8f2d21460) MSP API: 1.43
+
+board_name JHEF411
+manufacturer_id JHEF
+
+# resources
+resource BEEPER 1 C14
+resource MOTOR 1 A08
+resource MOTOR 2 A09
+resource MOTOR 3 A10
+resource MOTOR 4 B00
+resource MOTOR 5 B04
+resource MOTOR 6 NONE
```

Comment:
Please read and apply https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md to this - there should not be any lines that are defaults in here.

Reply:
Good point, fixed!